spec.md

```diff
@@ -864,6 +867,31 @@ message GetCapacityResponse {
 }
 ```
 
+#### `ProbeController`
+
+A Controller Plugin MUST implement this RPC call.
+The Plugin SHALL assume that this RPC will be executed on the controller which could be either a central service or a distributed one.
+The CO SHOULD call this RPC for the controller at which it wants to create volume.
+This RPC allows the CO to probe the readiness of the Plugin on the controller.
+The Plugin SHOULD verify if it has the right configurations to run the controller, and return a success if the validation succeeds.
```

Comment:
I'd bump this up to be the 2nd line of the paragraph

Reply:
s/configurations/configurations, devices, dependencies and drivers in order/
s/controller/controller service/